### ISSUE TITLE
Accessible ino-snackbar

### ISSUE THREAD

**Issue by Sl1nd** [inovex/elements#994]
According to W3C WAI [Alert Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) brief, important messages should be specified with `role` `alert`.
